providers/azure.go

```diff
@@ -360,6 +367,38 @@ func (p *AzureProvider) redeemRefreshToken(ctx context.Context, s *sessions.Sess
 	return nil
 }
 
+func (p *AzureProvider) getUser(ctx context.Context, accessToken string) (string, error) {
+	header := makeAzureHeader(accessToken)
+	endpoint := p.Data().ValidateURL.String()
+	if len(header) == 0 {
```

Comment:
Can this ever be empty?

Reply:
good note ... it seems that in this case there will always be at least the `Authorization` header, so I can delete it. I took inspiration from https://github.com/oauth2-proxy/oauth2-proxy/blob/master/providers/internal_util.go#L54 but now I see that it is there because of the fact that in some cases there can also be null.